### DISCUSSION TITLE
Related to Stack selection for My applicaction

### DISCUSSION THREAD

**Discussion by AmolMeshram-eaton** [openthread/openthread#9713] [Q&A] [answered]
Required information which stack to be used for creating open thread network and sending data to Leader or router from End point.
1:  Required open thread stack which will Exchange data within 500 msec or less, with packet size of 128 bytes. Nodes will transmit data to Border router or leader.
2:  Border router stack should be compatible with Bluetooth for commissioning purpose. (for network creation)

**Comment by AmolMeshram-eaton** [comment]
Hello, Which stack to be used for Router and End point please suggest.

**Comment by jwhui** [comment]
OpenThread supports all of the roles you are seeking to support.

**Reply by AmolMeshram-eaton** [reply]
There are some stacks available but which one to use for any specific please suggest:
1: Like RCP (Multiprotocol stack with OT +BLE ) for border router.
2: And For end point MTD stack.
Is it correct, Is it BLE can be use for commissioning through border router.

My setup is : Leader, Border router and some node devices.

**Comment by AmolMeshram-eaton** [comment]
Hello, I am using Silicon labs EFR32 boards for my product, It seems to exmaple codes available with them. Is there any suggestions pls update

**Comment by silabs-danielV** [comment, marked as answer]
Hello,

Please refer to this link for more information on how to build, flash, and run OpenThread example code on an EFR32: [OpenThread on EFR32](https://github.com/openthread/ot-efr32/blob/main/src/README.md#openthread-on-efr32).
